pkg/acs/lib/acscte/docte.c

```diff
@@ -276,7 +276,7 @@ int DoCTE (ACSInfo *acs_info, const bool forwardModelOnly) {
         unsigned nScaleTableColumns = N_COLUMNS_FOR_RAZ_CDAB_ALIGNED_IMAGE;
 
         /* Read the parallel CTE parameters */
-        trlmessage("(pctecorr) Collecting data for Correction Type: parallel.\n");
+        trlmessage("(pctecorr) Collecting data for Correction Type: parallel.");
```

Comment:
Apparently this inserts a blank line in the log of a regular run. I guess I will add it back in.

```suggestion
        trlmessage("(pctecorr) Collecting data for Correction Type: parallel.\n");
```